exp-player/addon/styles/components/isp-radio-group.scss

```diff
@@ -24,6 +24,11 @@
   margin-top: -15px;
 }
 
-.label-spacing {
-  max-width: 6%;
+.measure-four {
+  width: 6%;
```

Comment:
In this case, I added the specific class for measure four because it has 11 options, and needed a smaller width than the existing css classes. 

I added the `measure-six` class to format this question:  
![screen shot 2016-09-26 at 4 18 58 pm](https://cloud.githubusercontent.com/assets/6414394/18850515/e8b74e5e-8405-11e6-9aeb-330b38dc0252.png)

If I use the existing `format-label` class it will look like this: 
![screen shot 2016-09-26 at 4 22 23 pm](https://cloud.githubusercontent.com/assets/6414394/18850537/08dd3a2c-8406-11e6-9fa6-96b23bee5afe.png)

Personally, I like the top one better because it's more spaced out, which is why there is a custom css class.